client/reader/tag-stream/main.jsx

```diff
@@ -149,6 +149,8 @@ class TagStream extends Component {
 				showFollowInHeader={ true }
 				forcePlaceholders={ ! tag } // if tag has not loaded yet, then make everything a placeholder
 				streamHeader={ tagHeader }
+				showSiteNameOnCards={ false }
+				useCompactCards={ true }
```

Comment:
@Addison-Stavlo @eoigal You should be able to add the `useCompactCards={ true }` prop to the `Stream` component you are using for the Discover page to adopt this same layout.